blog/_posts/2023-03-23-rse-at-leeds.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: page
+title:  Opportunities with the Research Computing team
+date:   2023-03-23
+subheadline: "Recruitment"
+teaser: "
+ We'll soon be advertising some opportunities to join our team. Read more to find out what we're about!
+"
+author: Alex Coleman
+image:
+  thumb: HPC3small.jpg
+
+breadcrumb: true
+---
+
+We're very excited to announce we're growing the Research Computing team here at Leeds!
+
+We're growing the team over the next 12 months and adding more research software engineer roles to the team. 
+Research software engineers are a crucial part of Research IT here at the University of Leeds, delivering computational training to researchers, providing research software engineering consulting for research projects, supporting the University's high performance computing systems and developing our research computing community at the University. 
+We're currently a small, friendly team, and we come from a real variety of backgrounds including data scientists, web developers, and specialists in distributed graphics.
+To give you a flavour of what the team is about and why you should apply check out these reasons below:
+
+1. **Work on diverse and interesting problems**, a core part of what we do is research software engineering consulting where academics come to us to ask for support on research projects. This means you get the opportunity to work on interesting and varied research problems, helping you develop new skills and overcome fascinating challenges. 
+
+2. **Contribute to world-leading research**, the University of Leeds supports fantastic world-leading research and for many researchers software is at the heart of it. Being part of Research Computing gives you the opportunity to work directly with world-leading academics on their work but also contribute indirectly by providing training and guidance to researchers. This role is a fantastic opportunity to help contribute to, and accelerate work that can change the world.
+
+3. **Flexibility**
```

Comment:
In the above descriptions it is already evident how flexible our work is. Maybe it just needs to be said that the hours are flexible and we work from home? Something like:

```suggestion
3. **Flexibility**, in addition to flexibility in relation to daily activities, the possibility of working in the most diverse areas, and the opportunity of using different tools and methodologies, our group also offers great flexibility in relation to work arrangements, supporting flexible hours and working from home.
```